js/stdlib/ts/interfaces.ts

```diff
@@ -161,7 +162,7 @@ export interface Stdlib_User<Provider, ProviderEnv, ProviderName, Token, Contrac
   balanceOf: (acc: Account, token?: Token) => Promise<BigNumber>
   balancesOf: (acc: Account, tokens: Array<Token | null>) => Promise<Array<BigNumber>>
   minimumBalanceOf: (acc:Account) => Promise<BigNumber>
```

Comment:
These should be `Address`, not `Account`

Reply:
this is out of the scope of the pr; do you want me to change this here?

Reply:
I decided to just make these changes a different pr